src/main/java/eu/dissco/orchestration/backend/properties/S3Properties.java

```diff
@@ -0,0 +1,19 @@
+package eu.dissco.orchestration.backend.properties;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "s3")
+public class S3Properties {
```

Comment:
Are these secrets accessible to this service? I thought they were just configured in the export job namespace

Reply:
If you need to do additional config for the secrets pls add it in the name of the pr (e.g. !Secret Config changes) tyty